src/sinol_make/__init__.py

```diff
@@ -30,7 +32,7 @@ def configure_parsers():
     return parser
 
 
-def main():
+def _main():
```

Comment:
```suggestion
def main_exn():
```

Is there any Python conventions to indicate that a function might throw? I'm familiar with the `_exn` suffix in other languages, but I'm not sure how it is in Python.

Reply:
I didn’t find any, but I think we should add it here